### DISCUSSION TITLE
next/image Error: Invalid src prop

### DISCUSSION THREAD

**Discussion by Chandler-Zhu** [vercel/next.js#38281] [Help] [answered]
I tried to use next/image component in my project, but it keeps giving me a server error, I already changed my `next.config.js` file according to the doc but the app is still broken
![image](https://user-images.githubusercontent.com/61914365/177063198-172f9c8b-f4be-40c6-8b16-79e3bcdef6e9.png)
repro:  https://stackblitz.com/edit/nextjs-vjxdw5?file=pages/index.js

**Comment by Chandler-Zhu** [comment, marked as answer]
fixed after I restart my computer

**Reply by tofuwu-dev** [reply]
I guess you didnt restart your next app after configuring in next config js

**Reply by aynaever** [reply]
no, I just had the same problem, and I made sure many times to restart the next app, and also they told me in the log they restarted the server after I updated next.config.js. But it didn't get solved till I restarted the computer.

**Reply by bwrtu1** [reply]
is there actually a real solution for this ? I have the same problem but i couldn't solve it

**Reply by Subi** [reply]
Any updates , issue is happening for me. I restarted next server as well.

**Reply by Denis2626** [reply]
Thank you good sir
